functions.js

```diff
@@ -23,8 +24,185 @@ function close() {
+    'link to live version': 'https://demo.dewcis.com/ueab/',
+    'link to source':
+      'https://github.com/microverse-students/palindrome-richardoppiyo',
+  },
+  {
+    title: 'No.3 Keeping track of hundreds of components',
+    name: 'Multi-Post Stories 3',
+    description: 'Gain+Glory',
+    image: 'images/ueab.png',
+    technologies: {
+      tech1: 'Ruby on rails',
+      tech2: 'CSS',
+      tech3: 'JavaScript',
+      tech4: 'Html',
+    },
+    'link to live version': 'https://demo.dewcis.com/ueab/',
+    'link to source':
+      'https://github.com/microverse-students/palindrome-richardoppiyo',
+  },
+  {
+    title: 'No.4 Keeping track of hundreds of components',
+    name: 'Multi-Post Stories 4',
+    description: 'Gain+Glory',
+    image: 'images/ueab.png',
+    technologies: {
+      tech1: 'Ruby on rails',
+      tech2: 'CSS',
+      tech3: 'JavaScript',
+      tech4: 'Html',
+    },
+    'link to live version': 'https://demo.dewcis.com/ueab/',
+    'link to source':
+      'https://github.com/microverse-students/palindrome-richardoppiyo',
+  },
+  {
+    title: 'No.5 Keeping track of hundreds of components',
+    name: 'Multi-Post Stories 4',
+    description: 'Gain+Glory',
+    image: 'images/ueab.png',
+    technologies: {
+      tech1: 'Ruby on rails',
+      tech2: 'CSS',
+      tech3: 'JavaScript',
+      tech4: 'Html',
+    },
+    'link to live version': 'https://demo.dewcis.com/ueab/',
+    'link to source':
+      'https://github.com/microverse-students/palindrome-richardoppiyo',
+  },
+  {
+    title: 'No.6 Keeping track of hundreds of components',
+    name: 'Multi-Post Stories 4',
+    description: 'Gain+Glory',
+    image: 'images/ueab.png',
+    technologies: {
+      tech1: 'Ruby on rails',
+      tech2: 'CSS',
+      tech3: 'JavaScript',
+      tech4: 'Html',
+    },
+    livelink: 'https://demo.dewcis.com/ueab/',
+    sourcelink:
+      'https://github.com/microverse-students/palindrome-richardoppiyo',
+  }];
+
+const containerPopUp = document.querySelector('#mobile-pop-up');
+
+// To get the empty section to populate with the JS object
+const container = document.querySelector('#container-div');
+
+// To loop through my object elements and add them to the html section
+projectList.forEach((item) => {
+  const content = `
+   <section id="container">
+      <div class="container-image">
+        <img class="item-image" src="${item.image}" alt="ueab-university" />
+      </div>
+      <div class="container-title">
+        <h4>
+          ${item.name} <br/>
+          ${item.description}
+        </h4>
+      </div>
+      <div class="container-projects">
+        <ul class="container-projects-element">
+          <li>${item.technologies.tech1}</li>
+          <li>${item.technologies.tech2}</li>
+          <li>${item.technologies.tech3}</li>
+          <li>${item.technologies.tech4}</li>
+        </ul>
+      </div>
+      <button onclick=${'createModal()'} class="container-btn" type="button" name="button">
```

Comment:
- [ ] **[OPTIONAL]** Eslint is warning you about unused var because you are calling the function `createModal()` here as a string. Try removing the function from the template literal and make sure the function works, then see that the Eslint warning is gone.

```suggestion
      </div>
      <button onclick=createModal() class="container-btn" type="button" name="button">
```